W-17882282-view-schedule-permissions (#771)
mulesoft/docs-runtime-manager: cloudhub/modules/ROOT/pages/managing-schedules.adoc

You cannot manage CloudHub schedules using scripts.
Use the *Schedules* tab or the CloudHub API to manage CloudHub schedules.

[IMPORTANT]

> To view the application schedules, you must have the *Exchange Viewer* permission, in addition to the *Read Applications* permission.

Reviewer: ```suggestion
To view the application schedules, you must have the *Exchange Viewer* and *Read Applications* permissions.
```
